### DISCUSSION TITLE
`passthroughMode`

### DISCUSSION THREAD

**Discussion by elsaco** [microsoft/terminal#13133] [General]
Building latest code release (head d82af93) there's an advanced option to enable `passthroughMode`. How is this feature supposed to be used or tested? Thx!

**Comment by zadjii-msft** [comment]
This is a per-profile setting that is super experimental, so watch out, here be dragons. 

It changes the way that the backing console works to totally remove ConPTY from the equation. For client apps that are just writing to the console and reading input, this should improve performance by a bit. For things that are making more elaborate usage of the console APIs, there'll be quite a few compatibility issues, for sure. 

Something like WSL should do well in passthrough mode, while something like powershell or cmd will do worse. 

However, the Terminal's VT implementation is still quite a bit lacking, so it's going to need a lot of love before WSL really feels right. Hence why it's limited strictly to Dev builds (not even Preview), while we wait for that to get sorted.
